tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/mqtt/imported/MQTTTest.java

```diff
@@ -380,7 +384,8 @@ public void testMQTTPathPatterns() throws Exception {
          Message msg = connection.receive(5, TimeUnit.SECONDS);
          do {
             assertNotNull("RETAINED null " + wildcard, msg);
-            assertTrue("RETAINED prefix " + wildcard, new String(msg.getPayload()).startsWith(RETAINED));
+            String msgPayload = new String(msg.getPayload());
```

Comment:
One worrisome thing.  I think there's some interdependence in these tests.  I originally added my interceptor checks in a new test method.  When I did that, this test started failing.  It was receiving the messages created previously.  I suspect the server isn't getting cleaned up 100% correctly, or maybe not at all.